test/bluemix.space.test.js

```diff
@@ -11,8 +11,12 @@ const helper = new Helper('../src/scripts');
 const expect = require('chai').expect;
 const mockUtils = require('./mock.utils.cf.js');
 const mockESUtils = require('./mock.utils.es.js');
+<<<<<<< HEAD
```

Comment:
Looks like some of the merge conflicts made it into this diff. @aeweidne  Could you take a look at fixing this?